sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/interpreter/BeamSqlExpressionExecutor.java

```diff
@@ -33,8 +32,7 @@
   void prepare();
 
   /** apply transformation to input record {@link Row} with {@link BoundedWindow}. */
-  List<Object> execute(
-      Row inputRow, BoundedWindow window, ImmutableMap<Integer, Object> correlateEnv);
+  List<Object> execute(Row inputRow, BoundedWindow window, BeamSqlExpressionEnvironment env);
```

Comment:
The row and window are now in the env also so this is redundant, but I'd rather do the fancy regexing and whatnot separate from this functional PR.